tools/clang/test/HLSLFileCheck/pix/rawBufferStore.hlsl

```diff
@@ -1,4 +1,4 @@
-// RUN: %dxc -enable-16bit-types -Emain -Tcs_6_3 %s | %opt -S -hlsl-dxil-pix-shader-access-instrumentation,config=U0:2:10i0;.. | %FileCheck %s
+// RUN: %dxc -enable-16bit-types -Emain -Tcs_6_3 %s | %opt -S -hlsl-dxil-pix-shader-access-instrumentation,config=U0:2:10i0;0;00; | %FileCheck %s
```

Comment:
typo `0;0;0`. I think this works for the reasons I cited above where the first zero is read as a terminator (default) then the absence of an integer is read as 0, which leaves you with "0;00;" and no problems really. but resolving that default to zero when there is no '.' to end the old assignments will likely break this.

Reply:
I've made all these consistent